modules/apps/journal/journal-web/test/js/SaveButtons.test.js

```diff
@@ -232,4 +232,31 @@ describe('SaveButtons', () => {
 			screen.queryByText('please-enter-a-valid-date')
 		).not.toBeInTheDocument();
 	});
+
+	it('does not proceed if required fields validation fails', async () => {
+		global.Liferay.Form = {
```

Comment:
This test is fine, you can either mock this validator, or even try rendering a component with a required field. Both cases are valid 👏 

```
const renderComponentWithRequiredField = (props = DEFAULT_PROPS) => {
	return render(
		<>
			<div className="article-content-content" />
			<input required id={`${props.portletNamespace}workflowAction`} />
			<input id={`${props.portletNamespace}jakarta-portlet-action`} />
			<SaveButtons {...props} />
		</>
	);
};
```